LibraBFT/Impl/Handle.agda

```diff
@@ -102,46 +102,33 @@ module LibraBFT.Impl.Handle where
  ...| V v = processVote now v
  ...| C c = return unit            -- We don't do anything with commit messages, they are just for defining Correctness.
 
+ -- Translation to SystemModel
```

Comment:
This comment is a bit unclear.  Does it apply to the import, or more stuff below?  I don't even feel the need for a comment here, but I am not sure what this one says.